cineast-core/src/main/java/org/vitrivr/cineast/core/features/OCRSearch.java

```diff
@@ -25,3 +79,221 @@ protected String enrichQueryTerm(String queryTerm) {
     return queryTerm + "~1";
   }
 }
+
+class TesseractRecognizer {
+
+  public String recognizer(BufferedImage image){
+    Tesseract tesseract = new Tesseract();
+    tesseract.setDatapath("resources/OCRSearch/TesseractData/tessdata/");
+    tesseract.setLanguage("eng");
+    tesseract.setTessVariable("tessedit_char_whitelist", "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890");
+    tesseract.setPageSegMode(6);
```

Comment:
it would be nice to have a comment here on why pagesegmode & ocrenginemode were chosen they way they were